### PR TITLE
Add NCZ-3010-HA Hinge Sensor

### PR DESCRIPTION
**Pull request by brendanjerwin**
http://nycesensors.com/wp-content/uploads/2016/03/NCZ-3010-Feature-Sheet-2015-07-14.pdf
http://nycesensors.com/wp-content/uploads/2016/03/NCZ-3010-HA-HA-Install-Guide-2015-07-29.pdf

### CODE REVIEW

**Comment by brendanjerwin** [comment]
Are there pointers to where else I should edit to add documentation and such?

File: devices.js [hunk 1]
```diff
@@ -4533,6 +4533,20 @@ const devices = [
...
+        zigbeeModel: ['3010'],
+        model: 'NCZ-3010-HA',
+        vendor: 'Nyce',
+        description: 'Door hinge sensor',
+        supports: 'contact',
+        fromZigbee: [
+            fz.ignore_basic_report,
+            fz.ignore_genIdentify, fz.ignore_poll_ctrl,
+            fz.generic_battery, fz.ignore_iaszone_report,
+            fz.iaszone_occupancy_2, fz.iaszone_contact,
```
Comment (Koenkk) [inline devices.js]: `fz.iaszone_occupancy_2` can be removed I think?

**Comment by sjorge** [comment]
These look cool, should check if my doors are compatible. They seem nice and hidden!

**Comment by sjorge** [comment]
@brendanjerwin did you ever get this to work?

**Pull request closed without merging**